benchmark/parse_output.py

```diff
@@ -0,0 +1,163 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015--, The WGS-HGT Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+#
+# Parse output files of HGT tools.
+#
+
+import click
+import sys
+
+
+def parse_trex(input_f):
+    """ Parse output of T-REX version 3.6.
+
+    Parameters
+    ----------
+    input_f: string
+        file descriptor for T-REX output results
+
+    Returns
+    -------
+    number_of_hgts: string
+        number of HGTs reported by a tool, or NaN if an error occurred
+    """
+    string = 'hgt : number of HGT(s) found = '
+    for line in input_f:
+        if string in line:
+            return line.split(string)[1].strip()
+    return "NaN"
+
+
+def parse_rangerdtl(input_f):
+    """ Parse output of RANGER-DTL version 1.0.
+
+    Parameters
+    ----------
+    input_f: string
+        file descriptor for RANGER-DTL output results
+
+    Returns
+    -------
+    number_of_hgts: string
+        number of HGTs reported by a tool, or NaN if an error occurred
+    """
+    string = "The minimum reconciliation cost is: "
+    for line in input_f:
```

Comment:
If you mean that the function will return HGT info from the first occurrence of the line with `string` present, then yes. Since we're passing one tree for analysis, there should only be one occurrence of the line.